test/controllers/webhook_controller_test.exs

```diff
@@ -7,14 +7,9 @@ defmodule Bigben.WebhookControllerTest do
   alias Bigben.Build
 
   test "POST /webhook with a passing build, creates an entry", %{conn: conn} do
+  payload = ~s/{"id":112129326,"repository":{"id":1294851,"name":"findit"},"number":"84","status":0,"result":0,"status_message":"Passed","result_message":"Passed","started_at":"2016-02-23T01:20:00Z","finished_at":"2016-02-23T01:21:00Z","duration":60,"branch":"master"}/
```

Comment:
Two things:
1. How about breaking this by line so we can see it well?
2. What do you think about removing the non-important stuff like `name` `number`, etc? Basically remove all things that we don't test for later.